tools/packaging/kata-deploy/Dockerfile

```diff
@@ -4,19 +4,10 @@
 
 # Specify alternative base image, e.g. clefos for s390x
 ARG IMAGE
-FROM ${IMAGE:-registry.centos.org/centos}:7
-ARG KATA_ARTIFACTS=./kata-static.tar.xz
+FROM ${IMAGE:-docker.io/centos}:7
```

Comment:
What's the reason for switching this to docker.io?  registry.centros.org doesn't provide an ARM image?

This should be explained as part of the commit message.

Reply:
@fidencio I test it. registry.centos.org/centos is not a manifest tag, i.e. it only provides x86_64 under that tag. docker.io provide all archs, hence go with docker.io

Reply:
addressed with new commit message.